srcpkgs/pyliblo/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'pyliblo'
+pkgname=pyliblo
+version=0.10.0
+revision=1
+wrksrc="pyliblo-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Cython"
+makedepends="python3-devel liblo-devel"
+depends="liblo"
```

Comment:
SONAME: liblo.so.7 <-> liblo>=0.26_1

This was detected automatically, so the `depends` entry isn't necessary.